kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/converter/FitbitBreathingRateAvroConverter.java

```diff
@@ -58,23 +56,22 @@ protected Stream<TopicData> processRecords(FitbitRestRequest request, JsonNode r
 
         return iterableToStream(br)
             .filter(m -> m != null && m.isObject())
-            .flatMap(FitbitAvroConverter::iterableToStream)
             .map(tryOrNull(m -> parseBr(m, startDate, timeReceived),
                 (a, ex) -> logger.warn("Failed to convert breathing rate from request {}, {}", request, a, ex)));
     }
 
     private TopicData parseBr(JsonNode data, ZonedDateTime startDate, double timeReceived) {
-      Instant time = startDate.with(LocalDateTime.parse(data.get("dateTime").asText())).toInstant();
+      Instant time = LocalDate.parse(data.get("dateTime").asText()).atStartOfDay(ZoneOffset.UTC).toInstant();
```

Comment:
Does this have to be at start of day?

Reply:
The dateTime is in `YYYY-MM-DD` format and `atStartOfDay` get the 12.00 am timestamp.

Reply:
Oh ok I see, I didn't realise it didn't have the time and only the date.

Reply:
Oh ok I see, I didn't realise it didn't have the time and only the date.